en/guides/students/exercises/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "[nl] Oefeningen oplossen"
```

Comment:
Why was the nl tag removed? We put this in title for Dutch pages on the English part of the website.